src/main/plugins/org.dita.base/xsl/preprocess/maplinkImpl.xsl

```diff
@@ -535,8 +531,7 @@ See the accompanying LICENSE file for applicable license.
     <!-- child found tag -->
     <xsl:param name="found" select="true()" as="xs:boolean"/>
     <!-- If going to print, and @print=no, do not create links for this topicref -->
-    <xsl:if test="not(($FINALOUTPUTTYPE = 'PDF' or $FINALOUTPUTTYPE = 'IDD') and @print = 'no') and 
```

Comment:
Same test as above, this was equivalent to
`<xsl:if test="true() and not(@processing-role = 'resource-only') and $found">`